src/style.js

```diff
@@ -87,7 +87,7 @@ export function getVendorJsPropertyName(style, camelCase, opt_bypassCache) {
 
 /**
  * Sets the CSS style of the specified element with optional units, e.g. "px".
- * @param {!Element} element
+ * @param {Element} element
```

Comment:
Fixed many type errors at once :)

On Fri, Sep 9, 2016 at 3:39 PM, Dima Voytenko <notifications@github.com>
wrote:

> In src/style.js
> <https://github.com/ampproject/amphtml/pull/4908#discussion_r78259513>:
>
> > @@ -87,7 +87,7 @@ export function getVendorJsPropertyName(style, camelCase, opt_bypassCache) {
> >
> >  /**
> >   * Sets the CSS style of the specified element with optional units, e.g. "px".
> > - * @param {!Element} element
> > + * @param {Element} element
>
> Why no !?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ampproject/amphtml/pull/4908/files/9dc6e433e9326d421095feb7f85465681e43f296#r78259513>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAFeT0y_xOCDxgMXyOpFdru8dTFLa3p7ks5qoeAbgaJpZM4J5hjb>
> .
>